File: java/src/main/java/com/ellen/java/jmvp/base/BaseView.java
package com.ellen.java.jmvp.base;

public interface BaseView{
    /**
     * 被Presenter绑定时调用
     */
    void attachedByPresenter();

    /**
     * 被Presenter解绑时调用
     */
    void detachedByPresenter();
}

File: app/src/main/java/com/ellen/androidarchitecture/kmvp/base/BaseView.kt
package com.ellen.androidarchitecture.kmvp.base

/**
 * MVP中的V层
 */
interface BaseView{
    /**
     * 绑定时调用
     */
    fun attachedByPresenter(){}

    /**
     * 取消绑定时调用
     */
    fun detachedByPresenter(){}
}